docs/src/overview.md

```diff
@@ -1,11 +1,50 @@
 # [Overview](@id overview)
 
-# Overview of the BioJulia Ecosystem
+# BioJulia: Fast, open, easy software for biology
 
-BioJulia is an 
+> Note: This landing site is under extensive development and will receive
+> frequent updates. It is not in a ready state, and is published under GitHub
+> Pages only for testing purposes. 
 
-## 
+BioJulia is a passionate, community-led organization providing biology-related
+packages written in the [Julia programming language](https://julialang.org/).
+The organization offers a comprehensive, fully open-source ecosystem of both
+libraries that serve as essential building blocks for other packages as well as
+interactive tools for everyday tasks and workflows. 
 
-## Common Julia packages
+Biologists and other scientists are fully empowered by Julia to easily tackle
+domain-specific challenges, taking advantage of features including:
+* [Fully reproducible
+  environments](https://pkgdocs.julialang.org/v1/environments/) thanks to
+  Julia's built-in package manager
+* [Competitive performance](https://julialang.org/benchmarks/) that rivals that
+  of lower-level, more complex languages such as C and Fortran
+* [Unicode-based math symbol
+  support](https://docs.julialang.org/en/v1/manual/unicode-input/), [transparent
+  BLAS
+  integration](https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/#man-linalg),
+  and additional features for performing complex numerical operations   
+* [A batteries-included read-eval-print loop
+  (REPL)](https://docs.julialang.org/en/v1/stdlib/REPL/#The-Julia-REPL) for
+  interactive data exploration and prototyping
+* [Seamless
+  interoperability](https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/#Calling-C-and-Fortran-Code)
+  [(JLLs,](https://docs.binarybuilder.org/stable/#Project-flow)
```

Comment:
Maybe rephrase this - only experienced Julians will know what a JLL is. Maybe "Easily depend on and use executable programs written in other languages" or something